lib/galaxy/datatypes/text.py

```diff
@@ -84,6 +84,44 @@ def sniff_prefix(self, file_prefix: FilePrefix) -> bool:
         return False
 
 
+@build_sniff_from_prefix
+class Zarr(Text):
+    """Class describing a zarr dataset"""
+    
+    edam_format = "format_3915"
+    file_ext = "zarr"
+
+    def set_peek(self, dataset: DatasetProtocol, **kwd) -> None:
+        if not dataset.dataset.purged:
+            dataset.peek = "Zarr file"
+            dataset.blurb = nice_size(dataset.get_size())
+        else:
+            dataset.peek = "file does not exist"
+            dataset.blurb = "file purged from disk"
+    
+    def get_mime(self) -> str:
+        """Returns the mime type of the datatype"""
+        return "text/html"
```

Comment:
Does Zarr has a mimetype?

Reply:
I did not find anything specific mime type for zarr. In some documentation, the mime type was set to `application/zip` (see [pygeopi](https://docs.pygeoapi.io/en/stable/data-publishing/ogcapi-coverages.html)) but I don't think it makes much sense. Does it?

Reply:
I guess then I would remove it. html is a wrong mimetype I think,

Reply:
Can I remove the function get_mime completely?

Reply:
Sure. Please look at the other datatypes.